modules/gltf/src/lib/gltf-scenegraph.js

```diff
@@ -493,6 +493,8 @@ export default class GLTFScenegraph {
       case 'texcoord':
       case 'texcoords':
         return 'TEXCOORD_0';
+      case 'batchid':
```

Comment:
It is weird to start adding non-gltf attributes here. We can change this switch statement to use `toUpperCase()` instead.